protobuf/protoc-gen-gogrpcmock/plugin/plugin.go

```diff
@@ -131,26 +145,44 @@ func (g *grpcmock) generateMockMessage(msg *generator.Descriptor, inner, nullabl
 		fieldType, _ := g.GoType(msg, field)
 		repeated := field.IsRepeated()
 		nullable := gogoproto.IsNullable(field) && field.IsMessage()
-
-		if field.OneofIndex != nil {
-			// TODO: Deal with oneof
+		oneof := field.OneofIndex != nil
+
+		if oneof {
+			if oFields[*field.OneofIndex] == nil {
+				of := oneofField{
+					fieldName: fieldName,
```

Comment:
Would this field name be `could_be` in this case? I was thinking it should be `some_oneof`.

Reply:
Not easy to understand but `GetFieldName` will be `some_oneof ` and `GetOneOfFieldName` will be `could_be `